content/en/platform/corda/4.11/enterprise/release-notes-enterprise.md

```diff
@@ -17,6 +17,12 @@ weight: 10
 
 # Corda Enterprise Edition 4.11 release notes
 
+## Corda Enterprise Edition 4.11.1 release notes
+
+Interoperability fix between 4.11 and pre-4.11 nodes when sending/fetching transactions for new data type: `TRANSACTION_RECOVERY`.
```

Comment:
For consistency, can you use same format as 4.10.3, i.e. Upgrade Recommendation, Fixed Issues title etc, you can use same text as 4.10.3 for upgrade recommendation.

Reply:
Updated